### PR TITLE
Load ircb data from file

### PR DESCRIPTION
**Pull request by slick666**
Implemented solution to issue #25 (1st test). Renamed method create to network_create in networks.py from ircb cli
Fix network.py, loaddata.py accordingly to pep8 style
Fix typo (pyyalm to pyyaml) in line 9.

### CODE REVIEW

**Comment by slick666** [comment]
Seems like the squash and rebase was al that was left. Just did that and threw it up here.

**Comment by rtnpro** [comment]
@slick666 could you please update the commit message to:
`Load ircb data from file. Fixes #25`? The rest of the details can go in summary.

File: ircb/cli/network.py [hunk 1]
```diff
@@ -28,8 +28,8 @@ def network_cli():
               type=click.Choice(
                   ['CERT_NONE', 'CERT_OPTIONAL', 'CERT_REQUIRED']))
 @coroutinize
-def create(user, network_name, host, port, nick, realname, username, password,
-           usermode,ssl,ssl_verify):
+def network_create(user, network_name, host, port, nick, realname, username, password,
```
Comment (slick666) [inline ircb/cli/network.py]: @rtnpro is this the create() that you were looking to rename? What was the reason behind the name change?

Comment (rtnpro) [inline ircb/cli/network.py]: I'd prefer `create` since the module name does the work of namespacing.

**Comment by slick666** [comment]
ok, reverted name back to create. updated the commit message per @rtnpro, let meknow if there is anything else we need done.

File: ircb/cli/network.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ def network_cli():
                   ['CERT_NONE', 'CERT_OPTIONAL', 'CERT_REQUIRED']))
 @coroutinize
 def create(user, network_name, host, port, nick, realname, username, password,
-           usermode,ssl,ssl_verify):
+                   usermode, ssl, ssl_verify):
```
Comment (pypingou) [inline ircb/cli/network.py]: the over-indentation here isn't pep8 valid if you want to follow pep8 :)

File: requirements.txt [hunk 3]
```diff
@@ -13,4 +13,4 @@ aiohttp==0.20.2
 aiohttp-auth==0.1.1
 aiohttp-session[secure]==0.5.0
 aiozmq==0.7.1
-msgpack-python==0.4.7
\ No newline at end of file
+msgpack-python==0.4.7
```
Comment (pypingou) [inline requirements.txt:16]: Amusing, I can't spot the difference here :)

Comment (rtnpro) [inline requirements.txt:16]: Yeah! Is it a new line change?

File: ircb/cli/network.py (hunk 2 above)
Comment (slick666) [inline ircb/cli/network.py]: weird, I was pretty sure I ran pep8 on this, good catch

File: requirements.txt (hunk 3 above)
Comment (slick666) [inline requirements.txt:16]: it's because I added a blank line at the end of the file. I can't figure out why the github diff shows it this way

**Comment by rtnpro** [comment]
Looks good to me :+1: Merging.

**Pull request merged**